app/views/directives/bind-service/delete-binding-select-form.html

```diff
@@ -3,18 +3,18 @@ <h3 class="mar-top-none">
 </h3>
 <form name="ctrl.bindingSelection" class="mar-bottom-lg">
   <fieldset ng-disabled="ctrl.isDisabled">
-    <div ng-repeat="binding in ctrl.bindings" class="radio">
+    <div ng-repeat="binding in ctrl.bindings | orderBy: ctrl.firstAppForBindingName : binding" class="radio">
```

Comment:
Worries me that the ordering wasn't flipped. We should double check the sorting, but either way this should fix the immediate problem